deploy/helm/rawfile-localpv/templates/controller/statefulset.yaml

```diff
@@ -51,14 +51,23 @@ spec:
               value: {{ .Values.logLevel }}
             - name: LOG_FORMAT
               value: {{ .Values.logFormat }}
-            - name: PLUGIN_TYPE
+            - name: CSI_DRIVER__PLUGIN_TYPE
               value: controller
+            {{- if .Values.reservedCapacity }}
             - name: RESERVED_CAPACITY
               value: {{ .Values.reservedCapacity | toString | quote }}
+            {{- end }}
+            {{- if .Values.capacityOverride }}
             - name: CAPACITY_OVERRIDE
               value: {{ .Values.capacityOverride | toString | quote }}
-            - name: GRPC_WORKERS
+            {{- end }}
+            - name: CSI_DRIVER__GRPC_WORKERS
               value: {{ .Values.controller.grpcWorkers | toString | quote }}
+            - name: NODE_ID
```

Comment:
How come `NODE_ID` is showing up twice, here and then as `CSI_DRIVER__NODE_ID` ?